extras/build_hashes.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import os
+import os.path as pth
```

Comment:
Apparently you don't need to import `os.path` if you import `os`, I thought there was some weird reason you had to.